oaknational/Oak-Web-Application
# #LESQ-1000: Newsletter signup check box

## Description

Music year: 2016

- Adds form and controller to the onboarding page form
- Default selects newsletterSignUp to be true 
- Submits the true or false value to the data from object

## Issue(s)

AC for ticket has slightly changed with Hubspot intergration being extrapolated into separate ticket and Manual input of school not created as of yet

Fixes #LESQ-1000

## How to test

1. Go to https://deploy-preview-2683--oak-web-application.netlify.thenational.academy/onboarding
2. You should see onboarding form now has sign up to newsletter checkbox defaulted to true

## Screenshots

How it should now look:
![Screenshot 2024-08-09 at 12 04 52](https://github.com/user-attachments/assets/e23af6ca-ce9f-4cb9-8d5a-e62a031ae799)

## Checklist
